x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/snapshotlifecycle/history/SnapshotHistoryStoreTests.java

```diff
@@ -167,7 +169,12 @@ public static SnapshotLifecyclePolicy randomSnapshotLifecyclePolicy(String id) {
             randomAlphaOfLength(4),
             randomSchedule(),
             randomAlphaOfLength(4),
-            config);
+            config,
+            randomRetention());
+    }
+
+    private static SnapshotRetentionConfiguration randomRetention() {
```

Comment:
This is pretty nitpicky but this method can be reused between this and `SnapshotLifecyclePolicyMetadataTests` instead of copy/pasted. If the base is in core I think it could also be reused in `SnapshotLifecyclePolicyTests`.

Reply:
I put it in the `core` package and used it everywhere now